### ISSUE TITLE
Create a Status Report for Seraph

### ISSUE THREAD

**Issue by schwesig** [AICoE/seraph#26]
- [x] Meeting with @tumido 
- [x] Writing Status Report 1. draw
- [x] @tumido adds links to PRs and Issues
- [x] ask Pau about right structure and focus
- [x] Finalize status report
- [x] Send it to Pau

e.g.
Obstacles
Components
MUST: Open Source
What is needed for a MVP
never Authentication
never UI

What is important for first release

**Comment by schwesig** [comment]
## Titel
seraph

## Project Manager
@pgarciaq, @tumido

## Project Sponsor
Radek Vokál, @durandom 

## Project Team
@SamoKopecky, @mimotej, @schwesig 

## Executive Summary
Headline: Bringing the idea of insights upstream and to open source.
must: Open Source

Create an upstream managed platform for (insights) advisor in a more abstract version for different linux versions.

## Milestone Status Review
2021-10-18 Radek & Marcel Pre-RampUp, including Advisor PM Stefan, Platform PM Rob, ecosystem PM Megan
2021-11-02 Kick-Off Call (Radek, Marcel, Tom, Thorsten)
2021-11-11 Pau joined the regular meeting
2022-01-26 Bryan got included
2022-03-24 Decision to start Wintersleep for Seraph until further notice/priorization

## Components
### Deployed
[ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) service
platform mq
[puptoo](https://github.com/RedHatInsights/insights-puptoo)

### Partial Deployment
host inventory

## Missing
Advisor

## Obstacles
advisor is not open source
 --> missing access to images and repositories (private)
host inventory backend container rep is not public
 --> container file in repository, needs to be build by yourself | we created our own image)

## Needed next steps to create a MVP
- Advisor
- GUI
- Authentication
- Open Source

## Engineering summary:
We've ingested manifests built for Clowder which are not suitable for multitenant clusters and converted them into Kustomize. Additionally we've followed Operate First guidance for cluster scoped resources which require elevated cluster permissions. Each application/component resources are kept in their respective folders and cluster scoped resources are in `cluster-scope` folder.

### Applications deployed:
- Ingress (https://github.com/AICoE/seraph/pull/17)
- Puptoo (https://github.com/AICoE/seraph/pull/22)
- Platform-MQ (Using Strimzi operator, https://github.com/AICoE/seraph/pull/12)
- Host Inventory (partial, https://github.com/AICoE/seraph/pull/24, https://github.com/AICoE/seraph/pull/25, https://github.com/AICoE/seraph/pull/27)

Payloads get ingested and successfully propagate to Platform-MQ. Puptoo validates payloads. Host inventory is not ingesting validated payloads, this is not finished.

## Remaining components left to deploy:
- Advisor (not started, codebase is not opensource)
- Storage broker (https://github.com/AICoE/seraph/pull/23)

**Comment by schwesig** [comment]
yes, that's more or less it
and also what's not solved, i. e. what would be missing for Advisor to be usable (even minimally) by users at operate-first
i. e. if I wanted to continue with the project on my own until I can add one of my RHEL systems to Seraph and get some recommendation from Seraph, what's missing?
TL;DR: execution status of the project plan

**Comment by tumido** [comment]
Engineering summary:
We've ingested manifests built for Clowder which are not suitable for multitenant clusters and converted them into Kustomize. Additionally we've followed Operate First guidance for cluster scoped resources which require elevated cluster permissions. Each application/component resources are kept in their respective folders and cluster scoped resources are in `cluster-scope` folder.

Applications deployed:
- Ingress (https://github.com/AICoE/seraph/pull/17)
- Puptoo (https://github.com/AICoE/seraph/pull/22)
- Platform-MQ (Using Strimzi operator, https://github.com/AICoE/seraph/pull/12)
- Host Inventory (partial, https://github.com/AICoE/seraph/pull/24, https://github.com/AICoE/seraph/pull/25, https://github.com/AICoE/seraph/pull/27)

Payloads get ingested and successfully propagate to Platform-MQ. Puptoo validates payloads. Host inventory is not ingesting validated payloads, this is not finished.

Remaining components left to deploy:
- Advisor (not started, codebase is not opensource)
- Storage broker (https://github.com/AICoE/seraph/pull/23)

**Comment by schwesig** [comment]
feedback from @pgarciaq
all OK
closing issue

**Comment by schwesig** [comment]
@durandom
